api/Element.json

```diff
@@ -3457,32 +3453,24 @@
               }
             ],
             "safari": {
-              "version_added": true,
+              "version_added": "11.1",
               "partial_implementation": true,
               "notes": "Does not support the <code>subtree</code> option.",
               "flags": [
                 {
                   "type": "preference",
                   "name": "Web Animations"
-                },
-                {
-                  "type": "preference",
-                  "name": "CSS Animations via Web Animations"
                 }
               ]
             },
             "safari_ios": {
-              "version_added": true,
+              "version_added": "11.3",
```

Comment:
This data suggests that `element.getAnimations()` isn't enabled by default in Safari, but it is. Can you add another statement for that too? I've checked all "Web Animations" and this seems to be the only such mistake.